.github/workflows/terraform.yaml

```diff
@@ -0,0 +1,28 @@
+name: Check the Terraform Module
```

Comment:
Should we put this together with [other shared workflows](https://github.com/canonical/sdcore-github-workflows)?

Reply:
Yes, it's added.